src/jit/importer.cpp

```diff
@@ -7102,33 +7120,32 @@ GenTreePtr          Compiler::impFixupStructReturnType(GenTreePtr op, CORINFO_CL
 #if defined(_TARGET_X86_) || defined(_TARGET_AMD64_)
 
 #ifdef FEATURE_UNIX_AMD64_STRUCT_PASSING
-    // No VarArgs for CoreCLR.
-    assert(!info.compIsVarArgs); 
+    // No VarArgs for CoreCLR on x64 Unix
```

Comment:
On windows CoreCLR supports varargs.  It just that on Unix it wasn't implemented, because it is not needed and deprioritized.  The fact that it is inside #ifdef FEATURE_UNIX_AMD64_... makes it obvious but i made is explicit by adding a couple of words to comment.

@LLITCHEV - is there a work item to track vararg work for x64 unix?